### PR TITLE
**REFACTOR**

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1372704325).

**All commits made as part of refactor efforts**

### CODE REVIEW

File: src/query/bns/bns.query.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { useQueries, useQuery, UseQueryResult } from 'react-query';
+
+import { useApi, Api } from '@store/common/api-clients.hooks';
+import { BnsNamesOwnByAddressResponse } from '@stacks/blockchain-api-client';
+
+const STALE_TIME = 15 * 60 * 1000; // 15 min
+
+const bnsQueryOptions = {
+  keepPreviousData: true,
+  cacheTime: STALE_TIME,
```
Comment (fbwoolf) [inline src/query/bns/bns.query.ts:10]: Realized this is a bit confusing bc there are options for both `cacheTime` and `staleTime`.

**Comment by fbwoolf** [comment]
Just a heads up on this, I am going to merge in my refactor branch. When I was testing I encountered the contract call in the test-app is hitting a white screen for regtest. It is on this main refactor branch too. So, it must not be my changes that broke it. Just noting it here when it comes up during QA.

**Comment by fbwoolf** [comment]
> Just a heads up on this, I am going to merge in my refactor branch. When I was testing I encountered the contract call in the test-app is hitting a white screen for regtest. It is on this main refactor branch too. So, it must not be my changes that broke it. Just noting it here when it comes up during QA.

Confirmed this is on main:
Screen Shot 2021-10-18 at 3 06 23 PM

My branch actually removed `contractIntefaceResponseState` so that part of the error is gone.

**Comment by fbwoolf** [comment]
> 
> My branch actually removed `contractIntefaceResponseState` so that part of the error is gone.

I fixed the error related to the contract, but it is still hitting an error trying to fetch the balances here:
https://github.com/blockstack/stacks-wallet-web/blob/b2a4ac575ae4b0c9fec11f034a5cd8da7a559234/src/pages/transaction-signing/hooks/use-transaction-error.ts#L26

**Comment by kyranjamie** [comment]
Mergey?

**Review by beguene — APPROVED** [review]

**Review by fbwoolf — APPROVED** [review]

**Pull request merged**